packages/utils/logger/src/formats/pretty-print.ts

```diff
@@ -0,0 +1,49 @@
+import { format, Logform } from 'winston';
+
+import { isString } from 'lodash/fp';
```

Comment:
```suggestion
```

Reply:
will have to make that an error in the eslitn config in the next steps :)